service/app.js

```diff
@@ -0,0 +1,282 @@
+import express from 'express'
+import mongoose from 'mongoose'
+import cors from 'cors'
+
+import { Deck } from './models/Deck.js'
+import { userInfo} from './models/User.js'
+
+//start the server
+const app = express()
+const port = 8000
+
+// Connect to MongoDB
+const connectionString = `mongodb+srv://${process.env.DB_USER}:${process.env.DB_PASSWORD}@cluster0.mvxpr.mongodb.net/FlashCard?retryWrites=true&w=majority`
+try {
+  await mongoose.connect(connectionString)
+} catch (err) {
+  console.log('error ', err)
+}
+
+// Middleware
+
+const exampleMiddleware = (req, res, next) => {
+  console.log('example middleware')
+  next()
+}
+
+app.use(cors())
+app.use(express.json())
+app.use(exampleMiddleware)
+
+// Get  Routes
+
+app.get('/', (req, res) => {
+  res.send('Hello, world!')
+})
+
+app.get('/decks', (req, res) => {
+  res.sendStatus(404)
```

Comment:
What is the purpose of this route?

Reply:
These was unnecessary route, it is removed.